docs/source/conf.py

```diff
@@ -23,6 +23,8 @@
 
 # Add `aiida` to the path so it can be imported without installing it
 sys.path.append(os.path.join(os.path.split(__file__)[0], os.pardir, os.pardir))
+# Add custom extensions to path
+sys.path.append(os.path.abspath('./_extensions'))
```

Comment:
What if this is somehow used from a different directory?
I suggest you use the same approach as above (`os.path.split(__file__)[0]`, perhaps define a for it constant)